src/main/kotlin/no/nav/medlemskap/domene/Datagrunnlag.kt

```diff
@@ -40,4 +42,16 @@ data class Datagrunnlag(
     fun gyldigeOrgnummer(): List<String> {
         return arbeidsforhold.gyldigeOrgnummer(kontrollPeriodeForArbeidsforhold)
     }
+
+    fun gyldigeAaRegUtenlandsopphold(): List<String> {
```

Comment:
Det er ikke behov for disse metodene, siden du har lagt metodene i arbeidsforhold-klassen, noe som er veldig bra. Du kan bruke metodene rett fra listen med arbeidsforhold i evalueringroute-klassen. Se kommentar på EvalueringRoute.